docs/configure-artifact-repository.md

```diff
@@ -360,36 +364,38 @@ artifacts:
       container: my-container-name
       blob: path/in/container
       # If a managed identity has been assigned to the machines running the
-      # workflow (e.g., https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
+      # workflow (for example, https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
       # then useSDKCreds should be set to true. The accountKeySecret is not required
       # and will not be used in this case.
       useSDKCreds: true  
 ```
 
-In addition to managed identities, Argo workflows also support authentication using access keys and SAS tokens.
+### Using Azure Access Keys
 
-### Using Azure access keys
+You can also use an [Access Key](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal).
 
-1. Retrieve the blob service endpoint for the storage account. For example:
+1. Retrieve the blob service endpoint for the storage account.
 
     ```bash
     az storage account show -n mystorageaccountname --query 'primaryEndpoints.blob' -otsv
+    # https://mystorageaccountname.blob.core.windows.net
     ```
 
-2. Retrieve the access key for the storage account. For example:
+2. Retrieve the Access Key for the storage account.
 
     ```bash
-    az storage account keys list -n mystorageaccountname --query '[0].value' -otsv
+    ACCESS_KEY="$(az storage account keys list -n mystorageaccountname --query '[0].value' -otsv)"
     ```
 
-3. Create a Kubernetes secret to hold the storage account key. For example:
+3. Create a Kubernetes Secret to hold the storage account key:
 
     ```bash
+    export ACCESS_KEY=$(az storage account keys list -n mystorageaccountname --query '[0].value' -otsv)
```

Comment:
Since `ACCESS_KEY` is already retrieved in the previous step (after https://github.com/argoproj/argo-workflows/pull/13376#discussion_r1687076355), this is duplicative.
Also, `export` is not necessary as it's used directly in the same shell as a variable.

```suggestion
```